src/test/app/Offer_test.cpp

```diff
@@ -102,68 +134,85 @@ class Offer_test : public beast::unit_test::suite
+        env.fund (XRP (10000), alice, gw);
+        env.trust (USD (100), alice);
 
-        env (pay (gw, "alice", USD (50)));
+        env (pay (gw, alice, USD (50)));
 
-        auto const firstOfferSeq = env.seq ("alice");
-        Json::StaticString const osKey ("OfferSequence");
+        auto const firstOfferSeq = env.seq (alice);
 
-        env (offer ("alice", XRP (500), USD (100)),
-            require (offers ("alice", 1)));
+        env (offer (alice, XRP (500), USD (100)),
+            require (offers (alice, 1)));
 
-        BEAST_EXPECT(isOffer (env, "alice", XRP (500), USD (100)));
+        BEAST_EXPECT(isOffer (env, alice, XRP (500), USD (100)));
 
         // cancel the offer above and replace it with a new offer
-        env (offer ("alice", XRP (300), USD (100)), json (osKey, firstOfferSeq),
-            require (offers ("alice", 1)));
+        env (offer (alice, XRP (300), USD (100)),
+             json (jss::OfferSequence, firstOfferSeq),
+             require (offers (alice, 1)));
 
-        BEAST_EXPECT(isOffer (env, "alice", XRP (300), USD (100)) &&
-            !isOffer (env, "alice", XRP (500), USD (100)));
+        BEAST_EXPECT(isOffer (env, alice, XRP (300), USD (100)) &&
+            !isOffer (env, alice, XRP (500), USD (100)));
 
         // Test canceling non-existant offer.
-        env (offer ("alice", XRP (400), USD (200)), json (osKey, firstOfferSeq),
-            require (offers ("alice", 2)));
+        env (offer (alice, XRP (400), USD (200)),
+             json (jss::OfferSequence, firstOfferSeq),
+             require (offers (alice, 2)));
+
+        BEAST_EXPECT(isOffer (env, alice, XRP (300), USD (100)) &&
+            isOffer (env, alice, XRP (400), USD (200)));
+
+        // Test cancellation now with OfferCancel tx
+        auto const nextOfferSeq = env.seq (alice);
+        env (offer (alice, XRP (222), USD (111)),
+            require (offers (alice, 3)));
+
+        BEAST_EXPECT(isOffer (env, alice, XRP (222), USD (111)));
+
+        Json::Value cancelOffer;
+        cancelOffer[jss::Account] = alice.human();
+        cancelOffer[jss::OfferSequence] = nextOfferSeq;
+        cancelOffer[jss::TransactionType] = "OfferCancel";
+        env (cancelOffer);
+        BEAST_EXPECT(env.seq(alice) == nextOfferSeq + 2);
 
-        BEAST_EXPECT(isOffer (env, "alice", XRP (300), USD (100)) &&
-            isOffer (env, "alice", XRP (400), USD (200)));
+        BEAST_EXPECT(!isOffer (env, alice, XRP (222), USD (111)));
     }
 
-    void testTinyPayment ()
+    void testcaseTinyPayment ()
     {
         testcase ("Tiny payments");
 
         // Regression test for tiny payments
         using namespace jtx;
         using namespace std::chrono_literals;
-        auto const alice = Account ("alice");
-        auto const bob = Account ("bob");
-        auto const carol = Account ("carol");
-        auto const gw = Account ("gw");
+        auto const alice = Account {"alice"};
```

Comment:
Thought: we should consider having a set of well-known accounts that _all_ tests share, instead of constructing them all over the place. No change needed.